frappe/twofactor.py

```diff
@@ -225,7 +225,7 @@ def get_email_subject_for_2fa(kwargs_dict):
 
 def get_email_body_for_2fa(kwargs_dict):
 	'''Get email body for 2fa.'''
-	body_template = 'Enter this code to complete your login:<br><br> <b>{{otp}}</b>'
+	body_template = 'Enter this code to complete your login:<br><br> <span style="font-size:18px;font-weight:bold;">{{otp}}</span>'
```

Comment:
```suggestion
	body_template = _("Enter this code to complete your login:") + '<br><br> <span style="font-size:18px;font-weight:bold;">{{otp}}</span>'
```

Shouldn't this part of the string be translatable?

Reply:
@gavindsouza I do not think it makes much of a difference because it is always English, it does not consider user language here. Subject and other messages are translatable but they still appear in English.